src/views/Actors/components/ActorItem/ActorItem.tsx

```diff
@@ -792,3 +793,9 @@ const PopoverListLinksStyled = styled(PopoverListLinks)(({ theme }) => ({
     },
   },
 }));
+
+const ScopeChipStyled = styled(ScopeChip)(({ theme }) => ({
+  [theme.breakpoints.up('desktop_1024')]: {
+    height: 24,
+  },
+}));
```

Comment:
Just a doubt, no change request here: is ScopeChip used only here or in other places too? Because if it is the same component it should have the styles inside the component itself and we can just use it here. It looks like it is a design issue or we're adding extra padding/font sizes or something and that's why, maybe, we need to adjust the height

Reply:
Yes , its the same component, inside of this view the component have different style